### PR TITLE
Update prepublish.js

### PR DESCRIPTION
**Pull request by hwzhangd**
add shasum check

### CODE REVIEW

File: scripts/prepublish.js [hunk 1]
```diff
@@ -49,7 +49,8 @@ function validateFile(fileName) {
 
         res = `${sha512}  ${fileName}`;
     } else {
-        res = exec(`shasum -a 512 ${fileName}`).trim();
+        
+        res = exec(`command -v shasum >/dev/null 2>&1 && shasum -a 512 ${fileName} || sha512sum -a 512 ${fileName}`).trim();
```
Comment (jbienkowski311) [inline scripts/prepublish.js]: I don't think that `sha512sum` needs the `-a 512` parameter

Comment (hwzhangd) [reply scripts/prepublish.js]: yes, it needn't parameter

**Comment by hwzhangd** [comment]
changed sha512sum command. 
and for linux build it need gcc+ version bigger than 4.7.0

**Comment by jbienkowski311** [comment]
Fixes #234

**Pull request merged**